charts/hpe-flexvolume-driver/v3.1.0/templates/hpe-flexvolume-plugin.yaml

```diff
@@ -0,0 +1,216 @@
+ #### Flexvolume Driver ###
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hpe-flexvolume-driver
+  namespace: kube-system
+  labels:
+    k8s-app: hpe-flexvolume-driver
+spec:
+  selector:
+    matchLabels:
+      name: hpe-flexvolume-driver
+  template:
+    metadata:
+      labels:
+        name: hpe-flexvolume-driver
+    spec:
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      serviceAccountName: hpe-flexvolume-sa
+      containers:
+      - name: flexvolume
+        image: {{ .Values.flexVolumeDriverImage}}: {{- .Values.flexVolumeDriverTag}}
+        imagePullPolicy: "Always"
+        lifecycle:
+          preStop:
+            # create empty file to let plugin signal handler to perform cleanup of config/cert/dory files
+            exec:
+              command: [ "/bin/sh", "-c", "touch /etc/hpe-storage/remove" ]
+        {{- if eq .Values.flavor "rke"}}
+          postStart:
```

Comment:
using default values to install will rasing the following message, is a valid HPE backend server required for testing or a fake IP would also work? 
Exec lifecycle hook ([/bin/cp -a /etc/hpe-storage/nimble.json /var/lib/kubelet/volumeplugins/hpe.com~nimble/nimble.json]) for Container "flexvolume" in Pod "hpe-flexvolume-driver-fvq8s_kube-system(df35e7c8-0131-43a1-ad2f-d2f2a0a288ab)" failed - error: command '/bin/cp -a /etc/hpe-storage/nimble.json /var/lib/kubelet/volumeplugins/hpe.com~nimble/nimble.json' exited with 1: /bin/cp: cannot create regular file '/var/lib/kubelet/volumeplugins/hpe.com~nimble/nimble.json': No such file or directory , message: "/bin/cp: cannot create regular file '/var/lib/kubelet/volumeplugins/hpe.com~nimble/nimble.json': No such file or directory\n"

Reply:
@guangbochen valid backend ip is required as our driver will generate certificates and login to the backend. We have tested this chart from our end with RKE.

Reply:
Thanks can u please help to squash those commits and I will LGTM to merge, thanks.

Reply:
@guangbochen done..thanks.